patches/minecraft/net/minecraft/block/material/Material.java.patch

```diff
@@ -0,0 +1,125 @@
+--- ../src-base/minecraft/net/minecraft/block/material/Material.java
++++ ../src-work/minecraft/net/minecraft/block/material/Material.java
+@@ -9,8 +9,8 @@
+     public static final Material field_151576_e = (new Material(MapColor.field_151665_m)).func_76221_f();
+     public static final Material field_151573_f = (new Material(MapColor.field_151668_h)).func_76221_f();
+     public static final Material field_151574_g = (new Material(MapColor.field_151668_h)).func_76221_f().func_76225_o();
+-    public static final Material field_151586_h = (new MaterialLiquid(MapColor.field_151662_n)).func_76219_n();
+-    public static final Material field_151587_i = (new MaterialLiquid(MapColor.field_151656_f)).func_76219_n();
++    public static final Material field_151586_h = (new MaterialLiquid(MapColor.field_151662_n)).func_76219_n().setIsLiquid().setCanDrownEntity().setCanPushEntity().setCanBeAbsorbed().setCanFloatBoat().setCanWaterPlants().setCanSpawnWaterCreatures().setIsSwimmable();
++    public static final Material field_151587_i = (new MaterialLiquid(MapColor.field_151656_f)).func_76219_n().setIsLiquid();
+     public static final Material field_151584_j = (new Material(MapColor.field_151669_i)).func_76226_g().func_76223_p().func_76219_n();
+     public static final Material field_151585_k = (new MaterialLogic(MapColor.field_151669_i)).func_76219_n();
+     public static final Material field_151582_l = (new MaterialLogic(MapColor.field_151669_i)).func_76226_g().func_76219_n().func_76231_i();
+@@ -59,7 +59,7 @@
+ 
+     public boolean func_76224_d()
```

Comment:
What is this?

Reply:
Yes, according to MCPBot the func_76224_d is the name for isLiquid